src/components/Calculator.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Calculator.css';
+
+// Child component
+function Button({ value, className }) {
+  return <button type="button" className={className}>{value}</button>;
+}
+
+Button.propTypes = {
+  value: PropTypes.string.isRequired,
+  className: PropTypes.string,
+};
+
+// Default props for Button component
+Button.defaultProps = {
+  className: '',
+};
+
+// Calculator component
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="calculator__wrap">
```

Comment:
- [ ] You did a good job, however, let's talk about the design, I know took time to do that but I'll suggest you stick to the design given as it is [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A). The design will be changed next time, but let's first make it like that. Kind fix it 💪. 

> | | Design | | Your project | |
> |-|---------|-|--------|-|
> | |<img width="558" alt="calculator" src="https://github.com/ClarenceAtim/MathMagicians/assets/57408419/08471b25-f24c-4e4f-9075-0901d1fd634e">| |![Screenshot 2023-07-11 122601](https://github.com/ClarenceAtim/MathMagicians/assets/57408419/90d3a90a-9122-437b-aa71-0779c0d09f13)| |